src/fastapibootcamp/handlers/external.py

```diff
@@ -50,3 +76,190 @@ async def get_index(
         application_name=config.name,
     )
     return Index(metadata=metadata)
+
+
+# =============================================================================
+# Lesson 1: A simple GET endpoint.
+
+
+@external_router.get("/hello", summary="Get a friendly greeting.")
+async def get_greeting() -> str:
+    return "Hello, SQuaRE Services Bootcamp!"
```

Comment:
I think I agree with keeping this simple, but it could be a bit surprising that this handler still returns JSON, so the response will have quote marks around it. The simplest response from an HTTP perspective would require a bit more complexity in FastAPI to specify `response_class=PlainTextResponse`. Maybe that's worth mentioning in a comment?

Reply:
I didn't think about that, but mentioning this will definitely clarify things.